src/panels/config/automation/condition/types/ha-automation-condition-numeric_state.ts

```diff
@@ -1,78 +1,73 @@
-import "@polymer/paper-input/paper-input";
-import "@polymer/paper-input/paper-textarea";
+import "../../../../../components/ha-form/ha-form";
 import { html, LitElement } from "lit";
 import { customElement, property } from "lit/decorators";
-import "../../../../../components/entity/ha-entity-picker";
+import memoizeOne from "memoize-one";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import type { HaFormSchema } from "../../../../../components/ha-form/types";
 import { NumericStateCondition } from "../../../../../data/automation";
-import { HomeAssistant } from "../../../../../types";
-import { handleChangeEvent } from "../ha-automation-condition-row";
+import type { HomeAssistant } from "../../../../../types";
 
 @customElement("ha-automation-condition-numeric_state")
 export default class HaNumericStateCondition extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
-  @property() public condition!: NumericStateCondition;
+  @property({ attribute: false }) public condition!: NumericStateCondition;
 
   public static get defaultConfig() {
     return {
       entity_id: "",
     };
   }
 
+  private _schema = memoizeOne((entityId): HaFormSchema[] => [
+    { name: "entity_id", selector: { entity: {} } },
+    {
+      name: "attribute",
+      selector: { attribute: { entity_id: entityId } },
```

Comment:
One day I hope we can make ha-form fields "context aware", for example, we would tell this selector to use the entity from `entity_id` data field automatically.